### PR TITLE
[vstest]VS test for VOQ System ports

### PR DESCRIPTION
**Pull request by vganesan-nokia**
**- Why I did it**

To add VS test cases to test VOQ system ports feature implemented by PRs:

https://github.com/Azure/sonic-swss-common/pull/380
https://github.com/Azure/sonic-sairedis/pull/657
https://github.com/Azure/sonic-swss/pull/1431

**- How I did it**

This PR is an enabler of the SAI emulation for VOQ switch objects implemented by sonic-sairedis PRs 
https://github.com/Azure/sonic-sairedis/pull/698
https://github.com/Azure/sonic-sairedis/pull/699
https://github.com/Azure/sonic-sairedis/pull/700
https://github.com/Azure/sonic-sairedis/pull/701
https://github.com/Azure/sonic-sairedis/pull/702
https://github.com/Azure/sonic-sairedis/pull/703
and swss virtual chassis tests for VOQ switch objects
This is done by loading SYSTEM_PORT configs during bootup and providing correct core port index mapping
for local ports.

**- How to verify it**

sonic-swss/tests/pytest test_virtual_chassis.py will pass.

**- Which release branch to backport (provide reason below if selected)**

**- Description for the changelog**

Changes for supporting vstest for VOQ system ports. The changes include:
(1)Use of chassis_db.json is avoided since the SYSTEM_PORT is made
available in virtual chassis linecard's default_config.json which will
be loaded during bootup
(2)Core port index map file is introduced and is copied from virtual chassis
directory to hwsku direcory by start.sh
(3)vs sai profile is modified to include core port index map file name

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by vganesan-nokia** [comment]
retest vsimage please

File: platform/vs/docker-sonic-vs/start.sh [hunk 1]
```diff
@@ -75,7 +75,6 @@ supervisorctl start redis-server
 start_chassis_db=`sonic-cfggen -v DEVICE_METADATA.localhost.start_chassis_db -y $chassisdb_cfg_file`
 if [[ "$HOSTNAME" == *"supervisor"* ]] || [ "$start_chassis_db" == "1" ]; then
    supervisorctl start redis-chassis
-   python /usr/bin/chassis_db.py
```
Comment (saravanansv) [inline platform/vs/docker-sonic-vs/start.sh:78]: now that these files are no longer required, can the files chassis_db.py and chassis_db.json be removed ?

Comment (vganesan-nokia) [reply platform/vs/docker-sonic-vs/start.sh:78]: Yes. I'll remove them

Comment (vganesan-nokia) [reply platform/vs/docker-sonic-vs/start.sh:78]: Removed chassis_db.py here and chassis_db.json from https://github.com/Azure/sonic-swss/pull/1431

**Comment by vganesan-nokia** [comment]
retest vs please

**Review by saravanansv — APPROVED** [review]

**Comment by lguohan** [comment]
wonder why chassis_db.py is no longer needed?

**Review by lguohan — APPROVED** [review]

**Comment by vganesan-nokia** [comment]
> wonder why chassis_db.py is no longer needed?
This was a script that loaded system port config from a separate file (chassis_db.json). Since we moved the system port config to config db in config_db.json, the system pots configs are loaded as part of config_db.json load during boot up. So we do not need this anymore

**Pull request merged**